### PR TITLE
lib.image: add after option in perp services

### PR DESCRIPTION
**Pull request by eonpatapon**
`after` allows to control the starting order of services

### CODE REVIEW

**Comment by eonpatapon** [comment]
http://hydra.int0.aub.cloudwatt.net/eval/611

File: pkgs/lib/image.nix [hunk 1]
```diff
@@ -46,14 +46,23 @@ rec {
...
+  runOptions = { chdir ? "", command, ... }@args:
+    if chdir != "" then
+      args // { command = "exec runtool -c ${chdir} ${command}"; }
+    else
+      args // { command = "exec ${command}"; };
+
+  runOneShot = { oneShot ? false, name, command, ... }@args:
+    let
+      oneshot = pkgs.writeShellScriptBin "oneshot-${name}" ''
+        ${command} && touch /var/run/perp/${name}.success || touch /var/run/perp/${name}.fail
```
Comment (nlewo) [inline pkgs/lib/image.nix]: Is it not problematic to change the return code from the return code of `${command}` to 1 in case of failure?

Comment (nlewo) [reply pkgs/lib/image.nix]: Moreover, the file `${name}.fail` is never used.

Comment (eonpatapon) [reply pkgs/lib/image.nix]: Good catch, I will look at that.

I know ${name}.fail is not used, but could be in the future

**Comment by eonpatapon** [comment]
@nlewo ping

**Pull request merged**